pkg/odo/cli/catalog/describe/service.go

```diff
@@ -6,21 +6,32 @@ import (
 	"strings"
 
 	"github.com/olekukonko/tablewriter"
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/machineoutput"
 	"github.com/openshift/odo/pkg/odo/genericclioptions"
 	svc "github.com/openshift/odo/pkg/service"
+	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
 	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
 	ktemplates "k8s.io/kubectl/pkg/util/templates"
 )
 
 const serviceRecommendedCommandName = "service"
 
 var (
-	serviceExample = ktemplates.Examples(`  # Describe a service
-    %[1]s mysql-persistent`)
+	serviceExample = ktemplates.Examples(`  # Describe a service catalog service
+    %[1]s mysql-persistent
+	
+	# Describe a operator backed service
+	%[1]s 
+	`)
 
-	serviceLongDesc = ktemplates.LongDesc(`Describe a service type.
+	serviceLongDesc = ktemplates.LongDesc(`Describes a service type.
+	This command supports both service catalog services and operator backed services.
+	A user can describe an operator backed service by providing the full identifier for an Operand i.e. <service-type>/<operand-type> which they can find by running "odo catalog list services".
```

Comment:
s/operator/Operator 

Operand is a resource started from Operator, AFAIK. What we're describing here is more along the lines of `<operator_type>/<cr_name>`.

Reply:
done